opendut-carl/src/grpc/peer_manager.rs

```diff
@@ -233,7 +238,12 @@ mod tests {
     async fn test_successful_create_delete() -> Result<()> {
 
         let resources_manager = Arc::new(ResourcesManager::new());
-        let testee = PeerManagerFacade::new(Arc::clone(&resources_manager), Vpn::Disabled, Url::parse("https://example.com:1234").unwrap());
+        let testee = PeerManagerFacade::new(
+            Arc::clone(&resources_manager),
+            Vpn::Disabled,
+            Url::parse("https://example.com:1234").unwrap(),
+            "I'm a ca cert string".to_string()
```

Comment:
use `include_str!()` and include `resources/development/tls/insecure-development-ca.pem`